zipkin2/src/main/java/zipkin2/storage/QueryRequest.java

```diff
@@ -98,6 +98,22 @@ public int limit() {
     return limit;
   }
 
+  /** Parent service name to fetch for the link dependency */
+  @Nullable
+  public String parentServiceName(){
```

Comment:
From what I recall we decided to use a separate query request for this called `DependencyQueryRequest`.